.github/workflows/weekly.yaml

```
@@ -45,7 +45,7 @@ jobs:
     name: Update Showcase Sites
     if: github.repository_owner == 'withastro'
     # Puppeteer is breaking with ubuntu-24.04, so we’re still using 22 for this workflow
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Leave my Puppeteer runner alone renovate!

```suggestion
    runs-on: ubuntu-22.04
```